opc-ua-sdk/sdk-client/src/main/java/org/eclipse/milo/opcua/sdk/client/subscriptions/TransferMonitoredItemResponse.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 the Eclipse Milo Authors
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.milo.opcua.sdk.client.subscriptions;
+
+import java.util.List;
+
+import org.eclipse.milo.opcua.sdk.client.api.subscriptions.UaMonitoredItem;
+import org.eclipse.milo.opcua.stack.core.types.builtin.StatusCode;
+
+public class TransferMonitoredItemResponse {
+
+    private final List<UaMonitoredItem> monitoredItems;
+    private final List<StatusCode> statusCodes;
+
+    public TransferMonitoredItemResponse() {
```

Comment:
Do you need these zero-arg constructors? I'd like to delete them if not.

Reply:
Not at all. I add them out of habit from my c# days where some serializers needed them but would give cryptic errors when they were not there.